tfexec/test_helper.go

```diff
@@ -278,20 +300,20 @@ provider "aws" {
     iam = "%s"
   }
 }
-`, dir, endpoint, endpoint, endpoint, endpoint)
+`, endpoint, endpoint, endpoint)
 	return backendConfig
 }
 
 // SetupTestAccWithApply is an acceptance test helper for initializing a
 // temporary work directory and applying a given source.
-func SetupTestAccWithApply(t *testing.T, workspace string, source string) TerraformCLI {
+func SetupTestAccWithApply(t *testing.T, workspace string, source string, backendConfig []string) TerraformCLI {
```

Comment:
As mentioned above, this change should also be written directly in the specific test case without modifying the helper function.

Reply:
Revert and directly write backend in 232c8ceece6b0caf7386bc98b531c77a26b369fc